src/unix/linux_like/linux/musl/mod.rs

```diff
@@ -344,6 +344,75 @@ s! {
         pub len: ::__u32,
         pub options: ::__u32,
     }
+
+    // netinet/tcp.h
+
+    pub struct tcp_info {
+        pub tcpi_state: u8,
+        pub tcpi_ca_state: u8,
+        pub tcpi_retransmits: u8,
+        pub tcpi_probes: u8,
+        pub tcpi_backoff: u8,
+        pub tcpi_options: u8,
+        /*
+         * FIXME: when musl headers are more up to date
```

Comment:
```suggestion
         * FIXME(musl): when musl headers are more up to date
```

Just so we can grep these things easy at the musl update